layers/ntp_test.go

```diff
@@ -111,12 +110,12 @@ func test_01(t *testing.T) {
 		ExtensionBytes:     []byte{},
 	}
 
-	check_ntp("test_01", t, testPacketNTP, p_expected_ntp)
+	checkNTP("test01", t, testPacketNTP, pExpectedNTP)
```

Comment:
You probably don't need the "descriptions" now, since the tests will show up as TestNTPOne/Two/Three.

Are there more descriptive names we could give, something detailing for example the type of packet that's being parsed?